ddtrace/tracer/option.go

```diff
@@ -131,6 +134,10 @@ type config struct {
 
 	// enabled reports whether tracing is enabled.
 	enabled bool
+
+	// maxTagsHeaderLen specifies the maximum length of the X-Datadog-Tags header value.
+	// It defaults to defaultMaxTagsHeaderLen.
+	maxTagsHeaderLen int
```

Comment:
Since there is no other option (besides env.) to configure this, I think we can remove the field.